package.json

```
@@ -240,73 +240,74 @@
+		"docs:build": "npm-run-all docs:gen docs:blocks docs:ref docs:theme-ref",
 		"docs:gen": "node ./docs/tool/index.js",
-		"docs:build": "npm-run-all docs:gen api-docs:*",
+		"docs:ref": "node ./bin/api-docs/update-api-docs.js",
+		"docs:theme-ref": "node ./bin/api-docs/gen-theme-reference.js",
+		"env": "wp-env",
 		"fixtures:clean": "rimraf \"test/integration/fixtures/blocks/*.+(json|serialized.html)\"",
-		"fixtures:generate": "cross-env GENERATE_MISSING_FIXTURES=y npm run test-unit test/integration/full-content/ && npm run format test/integration/fixtures/blocks/*.json",
-		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
+		"fixtures:generate": "cross-env GENERATE_MISSING_FIXTURES=y npm run test:unit test/integration/full-content/ && npm run format test/integration/fixtures/blocks/*.json",
+		"fixtures:regenerate": "npm-run-all fixtures:clean fixtures:generate",
 		"format": "wp-scripts format",
-		"format-php": "wp-env run composer run-script format",
-		"lint": "concurrently \"npm run lint-lockfile\" \"npm run lint-js\" \"npm run lint-pkg-json\" \"npm run lint-css\"",
-		"lint-js": "wp-scripts lint-js",
-		"lint-js:fix": "npm run lint-js -- --fix",
-		"prelint-php": "wp-env run composer \"install --no-interaction\"",
-		"lint-php": "wp-env run composer run-script lint",
-		"lint-pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
-		"lint-lockfile": "node ./bin/validate-package-lock.js",
-		"lint-css": "wp-scripts lint-style \"**/*.scss\"",
-		"lint-css:fix": "npm run lint-css -- --fix",
-		"lint:md-docs": "wp-scripts lint-md-docs",
-		"native": "npm run --prefix packages/react-native-editor",
+		"format:php": "wp-env run composer run-script format",
+		"lint": "concurrently \"npm run lint:lockfile\" \"npm run lint:js\" \"npm run lint:pkg-json\" \"npm run lint:css\"",
+		"lint:css": "wp-scripts lint-style \"**/*.scss\"",
+		"lint:css:fix": "npm run lint:css -- --fix",
+		"lint:js": "wp-scripts lint-js",
+		"lint:js:fix": "npm run lint:js -- --fix",
+		"lint:lockfile": "node ./bin/validate-package-lock.js",
+		"lint:md:docs": "wp-scripts lint-md-docs",
+		"prelint:php": "wp-env run composer \"install --no-interaction\"",
+		"lint:php": "wp-env run composer run-script lint",
+		"lint:pkg-json": "wp-scripts lint-pkg-json . 'packages/*/package.json'",
+		"other:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
+		"other:changelog": "node ./bin/plugin/cli.js changelog",
+		"other:check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2 --ignore=@react-native-community/cli,@react-native-community/cli-platform-ios\" \"wp-scripts check-licenses --dev\"",
+		"other:check-local-changes": "node ./bin/check-local-changes.js",
+		"other:cherry-pick": "node ./bin/cherry-pick.mjs",
+		"other:distclean": "rimraf node_modules packages/*/node_modules",
+		"other:native": "npm run --prefix packages/react-native-editor",
+		"other:precheck-local-changes": "npm run docs:build",
```

That's tricky with `pre` and `post` scripts. It should always be at the start:
```suggestion
		"preother:check-local-changes": "npm run docs:build",
```

---

Fixed with bb6a5bb 👍